docs/docs/connected-accounts/README.md

```diff
@@ -7,11 +7,15 @@ For example, you can connect to Slack from Pipedream (via their OAuth integratio
 ```javascript
 import { WebClient } from '@slack/web-api';
 
-const web = new WebClient(auths.slack.oauth_access_token)
-return await web.chat.postMessage({
-  text: "Hello, world!",
-  channel: "#general",
-})
+defineComponent({
+  async run({ steps, $ }) {
```

Comment:
Need to rewrite this to use the `slack` app prop (see comments on the auth doc above)